### ISSUE TITLE
Row header border line of the DataGridView control has very low luminosity contrast ratio

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#5961]
.NET Core Version: 
* .NET Core SDK: 7.0.0-alpha.1.21512.1

Have you experienced this same bug with .NET Framework?: 
* Yes

**Repro steps:**

1. Create a `Form` with a `DataGridView`.
2. Add multiple columns to a `DataGridView`.
3. Add several rows to the `DataGridView`.
4. Set `DataGridView.RightToLeft = RightToLefYes`
5. Run the application.

**Actual behavior:**
Row header border line of the `DataGridView` control has very low luminosity contrast ratio.

`RightToLeft = Yes`:
![image](https://user-images.githubusercontent.com/23376742/137126388-4fa54fee-286d-4b22-93ed-2d6d54291cc0.png)

`RightToLeft = No`:
![image](https://user-images.githubusercontent.com/23376742/137126519-05e89ce3-c24e-452e-ba36-26a06f6b80e8.png)

**Expected behavior:**
Luminosity contrast ratio of the row header border line of the `DataGridView` control should be greater than 3:1.

**Comment by elachlan** [comment]
@Olina-Zhang I think this might have been resolved in another issue/pr. Can your team please re-test?

**Comment by Olina-Zhang** [comment]
It still repro in the latest .Net 9.0 SDK build: 9.0.100-alpha.1.24053.14.
![image](https://github.com/dotnet/winforms/assets/26474449/b4159d27-5f19-49b7-943b-24644d5e1cd9)

**Comment by Olina-Zhang** [comment]
Verified this issue in the latest .NET 9.0 RC1 build, it was fixed:
![image](https://github.com/user-attachments/assets/5b410646-7554-4e1c-affe-c501fc28f363)

**Comment by Olina-Zhang** [comment]
Verified it with .NET 9.0.100-rc.1.24422.10 test pass build that the issue has been fixed, which have the same results as above.

**Comment by Tanya-Solyanik** [comment]
Reverted this fix - https://github.com/dotnet/winforms/pull/12246

**Comment by Tanya-Solyanik** [comment]
The border line is not an active UI element, contrast requirements are applicable to actionable elements.
